components/style/properties/gecko.mako.rs

```diff
@@ -2353,6 +2458,22 @@ fn static_assert() {
 
     <%call expr="impl_app_units('_webkit_text_stroke_width', 'mWebkitTextStrokeWidth', need_clone=False)"></%call>
 
+    #[allow(non_snake_case)]
+    pub fn set__moz_tab_size(&mut self, v: longhands::_moz_tab_size::computed_value::T) {
```

Comment:
Note that `Number` and `Length` already `impl GeckoStyleCoordConvertible` and `Either` [could derive](https://dxr.mozilla.org/servo/rev/9eaf96b73716450caba5e85b9c5d3622242f2608/components/style/gecko/values.rs#35-48) from them.